app/middleware/verifyToken.js

```diff
@@ -1,23 +1,22 @@
 const jwt = require('jsonwebtoken');
+const { StatusCodes } = require('http-status-codes');
+
 const config = require('../../config');
 
 module.exports = (request, response, next) => {
     const token = request.token || null;
 
     if (!token) {
         return response
-            .status(403)
-            .json({ auth: false, message: 'No token provided.' });
+            .status(StatusCodes.UNAUTHORIZED)

```

Comment:
there was 403 so FORBIDDEN, Im not sure which one should be there

Reply:
if you want to go to endpoint you don't have permission but you are logged in then forbidden, if you are not logged in and you request for secured endpoint - then unauthorized